app/src/main/java/com/cappielloantonio/tempo/App.java

```diff
@@ -69,6 +71,10 @@ private static Subsonic getSubsonicClient() {
         String salt = Preferences.getSalt();
         boolean isLowSecurity = Preferences.isLowScurity();
 
+        if (Preferences.getLocale() != null) {
+            AppCompatDelegate.setApplicationLocales(LocaleListCompat.create(Preferences.getLocale()));
```

Comment:
I would assume that is not the right place to put this...

Reply:
Nope, but it's okay. Don't worry!